packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/price/price.component.js

```diff
@@ -0,0 +1,15 @@
+import controller from './price.controller';
```

Comment:
This component needs to be moved to modules folder as commented by Fred and Jeremy earlier. And also needs to be merged with exiting price component that Jeremy has developed.

Reply:
Yes you can use https://github.com/ovh/manager/tree/develop/packages/manager/modules/catalog-price it has been merged !

Reply:
This is done